### ISSUE TITLE
Detect `double parens` in function call argument

### ISSUE THREAD

**Issue by MKowalski8** [software-mansion/cairo-lint#337]
### Problem

The current implementation of `double_parens` does not detect the following case: 
``` cairo
fn foo(a: i32) -> i32 {
    0
}

fn main() {
    foo((0));
}
```
This case should be detected and then replaced with `foo(0)`

### Notes

In cases where we have more than double `foo(((0)))`, the situation is detected correctly.
